crates/wasmparser/tests/legacy-exception.rs

```diff
@@ -0,0 +1,26 @@
+// Tests for validate legacy-exception-handling
+
+#[test]
+#[cfg(feature = "legacy-exception-handling")]
+fn check_legacy_exception_handling_opcodes() {
+    let _binary_data = [0, 97, 115, 109, 1, 0, 0, 0, 1, 4, 1, 96, 0, 0, 3, 2, 1, 0, 13, 3, 1, 0, 0, 10, 21, 1, 19, 0, 6, 64, 6, 64, 6, 64, 8, 0, 25, 9, 0, 11, 24, 0, 7, 0, 11, 11];
+    let _text_format_data = "
+      (module
+        (type (;0;) (func))
+        (func (;0;)     (type 0)
+          try ;; label = @1
+            try ;; label = @2
+              try ;; label = @3
+                throw 0
+              catch_all
+                rethrow 0 (;@3;)
+              end
+            delegate 0 (;@2;)
+          catch 0
+          end
+        )
+        (tag (;0;) (type 0))
+      )";
+
+    // TODO: Use `_binary_data` or `_text_format_data`
+}
```

Comment:
I'd recommend adding this tests as `tests/local/legacy-exceptions.wast` where you could add a variety of modules as necessary.

Reply:
I don't fully understand this, I will take a closer look at it and let you know or ask for help.

Reply:
Updated ✅